tests/docker-compose.webhook.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.3'
+
+services:
+  webhook_receiver:
+    image: python:3.9-slim
+    restart: always
+    expose:
+      - "2020"
```

Comment:
Better to define the port as an environment variable. Thus you don't need to specify 2020 in multiple places.

Reply:
Declared this PORT `.env` file